Organize test categories, add a new code

Jace0615 opened the pull request:
1. Organize test categories
2. Add a new code

<!-- This is an auto-generated comment: release notes by coderabbit.ai -->
## Summary by CodeRabbit

- **새로운 기능**
	- "Agit" 기능에 대한 종합적인 엔드 투 엔드 테스트 스위트 추가.
	- 사용자 프로필 기능에 대한 자동화된 테스트 스위트 추가.
	- NFT 기능에 대한 새로운 테스트 스위트 추가.
	- 컬렉션 관련 사용자 상호작용을 테스트하는 새로운 테스트 추가.

- **버그 수정**
	- 기존의 제안 관련 테스트 스위트 삭제.

- **문서화**
	- 테스트 스위트의 구조와 흐름에 대한 설명 추가.
<!-- end of auto-generated comment: release notes by coderabbit.ai -->

Jace0615 committed:
New test.
Follow : follow a agit, follow a collection

Jace0615 committed:
added new code, changes some code

Jace0615 committed:
Modify code content
1. Reorganize categories
2. Add and modify code

Jace0615 committed:
Organized a test categories
- add 'collection', 'nft'
- remove 'follow', 'proposal'

hackartists commented on Makefile:
```diff
@@ -11,4 +11,4 @@ node_modules:
 
 test: node_modules
-	$(BUILD_ENV) npx playwright test
+	$(BUILD_ENV) npx playwright test --ui
```
@Jace0615  이거는 --ui 빼는게 나을듯 합니다. 이 커맨드로 Github action 이 돌거든요

hackartists commented on tests/agit.spec.js:
```diff
@@ -0,0 +1,160 @@
+    await page.getByText("DAO", { exact: true }).click();
+    await page.waitForTimeout(latency);
+    await page.getByText("DAO 테스트", { exact: true }).click();
+    await page.waitForTimeout(latency);
+    await page.locator('textarea[placeholder="댓글을 남겨주세요."]').fill("test");
+    await page.waitForTimeout(latency);
+    await page.press('textarea[placeholder="댓글을 남겨주세요."]', 'Enter');
+    await page.waitForTimeout(latency);
+    });
+    
+    test("click a like to proposal", async ({ page }) => {
+        await page
+        .locator('xpath=//*[@id="main"]/div[1]/div[4]/header/div/div[2]/div')
+        .click();
+        await page.waitForTimeout(latency);
+        await page.getByText("My Agit", { exact: true }).click();
+        await page.waitForTimeout(latency);
+        await page.getByText(agit_name, { exact: true }).click();
+        await page.waitForTimeout(latency);
+        await page.getByText("DAO", { exact: true }).click();
+        await page.waitForTimeout(latency);
+        await page.getByText("DAO 테스트", { exact: true }).click();
+        await page.waitForTimeout(latency);
+        await page.locator('svg.fill-gray-600').click();
+        await page.waitForTimeout(latency);
+    });
+
+    test("follow a agit", async ({ page }) => {
+        await page
+            .locator('xpath=//*[@id="main"]/div[1]/div[4]/header/div/div[2]/div')
+            .click();
+            await page.waitForTimeout(latency);
+            await page.getByText('My Agit', { exact: true }).click();
+            await page.waitForTimeout(latency);
+            await page.getByText(agit_name, { exact: true }).click();
+            await page.waitForTimeout(latency);
+            await page.getByRole("button", { name: "Follow" }).click();
+            await page.waitForTimeout(latency);
+        });
+    });
```
@Jace0615  여기에서 추천해주는거처럼 실패시에 스크린샷 저장해주는건 알아두면 좋을듯 하네요.

victor138128 commented on tests/create.setup.js:
```diff
@@ -8,6 +8,7 @@ import {
 } from "./constants";
 import path from "path";
 
+let agit=agit_name;
```
해당 변수는 이제 제거해도 될거 같아요!

victor138128 commented on Makefile:
```diff
@@ -11,4 +11,4 @@ node_modules:
 
 test: node_modules
-	$(BUILD_ENV) npx playwright test
+	$(BUILD_ENV) npx playwright test --ui
```
뒤에 --ui 제거 부탁드립니다!
ui 사용하시려면 14라인의 --ui는 제거하고, 14번째 라인 바로 밑에 test-ui를 하나 만드셔서 `make test-ui` 로 돌리셔도 될거 같습니다.

```
test-ui: node_modules
	$(BUILD_ENV) npx playwright test --ui
```

victor138128 reviewed (approved):
확인했습니다!

Jace0615 committed:
code changes
npx playwright test --ui -> npx playwright test
delete 'let agit=agit name;' code
